package.json

```diff
@@ -12,7 +12,8 @@
     "start-ts": "ts-node -r dotenv/config src/index.ts",
     "start-test": "ts-node -r dotenv/config src/utils/test.ts",
     "start": "node -r dotenv/config dist/index.js",
-    "build": "tsc --outDir dist"
+    "build": "tsc --outDir dist",
+    "code-review": "yarn lint && yarn test"
```

Comment:
we must define "test" script :)

Reply:
I've already added the "test" changes in this pull request -> https://github.com/pagopa/change-data-capturer-ms/pull/2

Reply:
We should provide test in this pull request ( so we can test pipeline directly in this PR )

Reply:
Ok fixed. The pipeline is green now.